client/css/style.css

```diff
@@ -1046,7 +1046,7 @@ kbd {
 	color: #50a656;
 }
 
-#chat .user:hover {
+.chat .user:hover {
```

Comment:
After this change, only nicks in the message list window will change opacity on hover, since hovering on the userlist adds a background instead, like our dropdowns.